core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/locks/OperationLock.java

```diff
@@ -33,8 +34,8 @@ public class OperationLock extends AbstractOperationLock {
 	 * @param id the lock identifier
 	 * @param target the lock target (may not be {@code null})
 	 */
-	public OperationLock(final int id, final Lockable target) {
-		super(id, target);
+	public OperationLock(final int id, final Date creationDate, final Lockable target) {
```

Comment:
Wouldn't it be better to use a LocalDate type here?

Reply:
Date includes the time aspect as well, not just a date, so LocalDateTime would be better, but that requires a larger refactor. Keep it as is for now.